src/styles/globals.scss

```diff
@@ -1,24 +1,15 @@
-@import "./components/home";
-@import "./components/header";
-@import "./elements/ul";
-@import "./components/componentCard";
+@import './components/home';
+@import './components/header';
+@import './elements/ul';
+@import './components/componentCard';
+@import './components/component-grid';
 
 html,
 body {
   padding: 0;
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    "Segoe UI",
-    Roboto,
-    Oxygen,
-    Ubuntu,
-    Cantarell,
-    "Fira Sans",
-    "Droid Sans",
-    "Helvetica Neue",
-    sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell,
+    'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
```

Comment:
I think if you run Bennett's `npm run lint` you're going to get flagged for all of the single quotes, and not having the font names on single lines.

Reply:
I'll send that up in the next commit though.

Reply:
Yep, I think all of the build failures are just due to linting errors.

<img width="971" alt="Screen Shot 2022-10-14 at 12 26 22 PM" src="https://user-images.githubusercontent.com/2685062/195898879-6cf3bbd1-e882-4e2b-a44d-f19306a832e4.png">